packages/react-components/src/components/table-output-component.tsx

```diff
@@ -38,7 +41,8 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
     private fetchColumns = true;
     private columnArray = new Array<any>();
     private pagination = true;
-    private paginationPageSize = 500000;
+    private paginationPageSize = 250000;
```

Comment:
Should this size change be part of this PR? I mean, versus #867 which already applies this fix?

Reply:
Indeed. Thanks. Fixed.